scapy/modules/krack/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Module implementing Krack Attack on client, as a custom WPA Access Point
+
+More details on the attack can be found on https://www.krackattacks.com/
+"""
```

Comment:
Could you add in the docstring the comprehensive process on "how to make this code work"? Using `load_module("krack")` rather than an `import` statement.